auth/src/main/res/values/styles.xml

```diff
@@ -199,13 +199,13 @@
     </style>
 
     <style name="FirebaseUI.Button.AccountChooser.EmailButton">
-        <item name="android:drawableLeft">@drawable/fui_ic_mail_white_24dp</item>
+        <item name="drawableStartCompat">@drawable/fui_ic_mail_white_24dp</item>
```

Comment:
Pre-L this will crash with a res not found exception if we don't manually get the drawable with `AppCompatResources.getDrawable`.